vggish_audio_segmenter.py

```diff
@@ -1,31 +1,116 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from typing import Optional
+from typing import Optional, Iterable
 import numpy as np
 
-from jina import Executor, DocumentArray, requests
-
+from jina import Executor, Document, DocumentArray, requests
+from vggish import vggish_input, vggish_params
 
 class VGGishSegmenter(Executor):
     """
-    Encode the Document blob into embedding.
+    Segment the Document blob into audio regions.
 
     :param embedding_dim: the output dimensionality of the embedding
     """
 
-    def __init__(self, embedding_dim: int = 128, *args, **kwargs ):
+    def __init__(self, sampling_factor: int=2, chunk_duration: int = 10,
+                sample_rate: int = 44100, default_traversal_paths: Iterable[str] = ['r'], *args, **kwargs ):
         super().__init__(*args, **kwargs)
-        self._dim = embedding_dim
+        self.sampling_factor = sampling_factor # i.e. the n in sampling notation s(nT)
+        self.chunk_duration = chunk_duration
+        self.sample_rate = sample_rate
```

Comment:
`sample_rate` should not be fixed in the `__init__` because each audio file has a different sample rate. I suggest putting this into `tags['sample_rate']`

```suggestion
        self.sample_rate = sample_rate
```

Reply:
Makes sense !